composer.json

```diff
@@ -22,23 +22,22 @@
         }
     ],
     "require": {
-        "php": "^7.1",
-        "symfony/config": "^4.4",
-        "symfony/dependency-injection": "^4.4",
-        "symfony/http-foundation": "^4.4",
-        "symfony/http-kernel": "^4.4",
+        "php": "^7.2",
+        "symfony/config": "^4.4 || ^5.0",
```

Comment:
Symfony 5.0 is unsupported: https://symfony.com/releases/5.0

Reply:
Fixed